opennms-doc/guide-admin/src/asciidoc/text/flows/classification-engine.adoc

```diff
@@ -51,52 +59,41 @@ A rule has the following fields:
 | The ip protocol of the flow must match this criteria.
 |===
 
-Even if all fields (besides `name`) are optional, at least one of them must be defined to be considered a valid rule.
-A list of pre-defined rules already exist.
+Even if all fields (besides `group`, `position` and `name`) are optional, at least one of them must be defined to be considered a valid rule.
+
+A list of pre-defined rules already exist in the group `pre-defined`.
 The pre-defined rules are inspired by the https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml[IANA Service Name and Transport Protocol Port Number Registry].
 New rules can be defined using the _Classification UI_ which can be found in the Admin Menu: *Admin -> Configure OpenNMS -> Manage Flow Classification*
 
 ==== Omnidirectional Rules
 Rules can be marked as omnidirectional which additionally evaluates the rules with interchanged endpoint addresses and ports.
-This is helpfull if traffic related to a matching classification should be classified the same way.
+This is helpful if traffic related to a matching classification should be classified the same way.
 
-==== Rule Priority
-User-defined rules always have a higher priority than the pre-defined rules.
-For example, if the user defines a new rule, `http` with a `dstPort` of `8980` that rule has a higher priority than the pre-defined rule `www-alt`.
+==== Rule Groups
+Rules live within a rule group. New groups can be added, edited and deleted via the _Classification UI_.
+The `pre-defined` group is read only. It (and it's rules) can not be altered.
```

Comment:
Can we add a [line break](https://xkcd.com/1285/) after a sentence? It makes it easier to git diff text. Reasons are here: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line